### PR TITLE
DQMEDAnalyzer skeleton

### PR DESCRIPTION
**Pull request by rovere**
Since I'm in the process of writing few DQM/Validation modules, I decided to prepare 2 skeletons to create them easily, one based on using the DQM as stream modules, the other using the recently introduced DQMGlobalEDAnalyzer with ConcurrentME.
I also fixed the EDAnalyzer skeleton to default it to create a working example based on tracks.

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: FWCore/Skeletons/scripts/mkdqmedanalyzer [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+#pylint: disable-msg=
+"""
+File       : mkdqmedanlzr
+Author     : Marco Rovere
```
Comment (Dr15Jones) [inline FWCore/Skeletons/scripts/mkdqmedanalyzer]: I doubt you meant to be the autor for ever single one of these :)

Comment (Dr15Jones) [reply FWCore/Skeletons/scripts/mkdqmedanalyzer]: Ah, I miss read, this is not the skeleton itself but is the generator. Sorry about that.

File: FWCore/Skeletons/scripts/mkTemplates/EDAnalyzer/EDAnalyzer.cc [hunk 2]
```diff
@@ -105,23 +104,23 @@ __class__::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
...
-@example_track    iEvent.getByLabel(trackTags_,tracks);
-@example_track    for(TrackCollection::const_iterator itTrack = tracks->begin();
-@example_track        itTrack != tracks->end();                      
-@example_track        ++itTrack) {
-@example_track       int charge = 0;
-@example_track       charge = itTrack->charge();  
-@example_histo       histo->Fill( charge );
-@example_track    }
+    Handle<TrackCollection> tracks;
+    iEvent.getByLabel(trackTags_,tracks);
```
Comment (Dr15Jones) [inline FWCore/Skeletons/scripts/mkTemplates/EDAnalyzer/EDAnalyzer.cc]: We should really change this to be a `getByToken` instead.

File: FWCore/Skeletons/python/cms.py [hunk 3]
```diff
@@ -48,6 +48,17 @@ def config(tmpl, pkg_help, tmpl_dir):
     kwds['tmpl_etags'] = etags
     return kwds
 
+def config_with_parser(tmpl, args, tmpl_dir):
```
Comment (Dr15Jones) [inline FWCore/Skeletons/python/cms.py]: please document the purpose of this function

File: FWCore/Skeletons/scripts/mkTemplates/EDAnalyzer/EDAnalyzer.cc (hunk 2 above)
Comment (rovere) [reply FWCore/Skeletons/scripts/mkTemplates/EDAnalyzer/EDAnalyzer.cc]: Indeed, thanks, I blatantly did cut and paste from old code. Will fix it.

File: FWCore/Skeletons/python/cms.py (hunk 3 above)
Comment (rovere) [reply FWCore/Skeletons/python/cms.py]: Sure, will add proper documentation.

**Comment by Dr15Jones** [comment]
+1

**Comment by rovere** [comment]
Any hits what exactly went wrong?

**Comment by Dr15Jones** [comment]
I've seen this same failure appear on other pull request tests today. I haven't figured out why.

**Comment by Martin-Grunewald** [comment]
please test

**Pull request merged**